server/src/main/xsd/gateway-config-201409.xsd

```diff
@@ -476,7 +476,7 @@
                      type="gateway:CollapsedString"/>
 
             <element maxOccurs="unbounded" minOccurs="0" name="accept"
-                     type="anyURI"/>
```

Comment:
I think most schema processors just don't do much processing (perhaps only invalid chars)
Are you running into any issues here ?